ament_copyright/ament_copyright/parser.py

```diff
@@ -44,6 +45,7 @@ def __init__(self, filetype, path):
         self.path = path
         self.exists = os.path.exists(path)
         self.content = None
+        self.license_identifier = UNKNOWN_IDENTIFIER
```

Comment:
See previous comment why I don't think the base class should have that attribute.